### ISSUE TITLE
Zentyal 5.1 : DHCP marked as stopped but really is running

### ISSUE THREAD

**Issue by doudz** [zentyal/zentyal#1820]
On dashboard, DHCP service is marked as stopped but really it's running
![image](https://user-images.githubusercontent.com/1262548/45409808-7e846b80-b670-11e8-95a4-62a3b6a0e71c.png)

```
ps aux|grep dhcp
dhcpd    32312  0.0  0.1  36088 14132 ?        Ss   09:43   0:00 dhcpd -user dhcpd -group dhcpd -f -4 -pf /run/dhcp-server/dhcpd.pid -cf /etc/dhcp/dhcpd.conf
cat /run/dhcp-server/dhcpd.pid
32312
```

**Comment by Neustradamus** [comment]
@jacalvo have you seen?

**Comment by jacalvo** [comment]
Can't reproduce this, please provide the detailed steps you follow to reach that situation, ideally from a fresh install. Thanks!

**Comment by jacalvo** [comment]
Could you check the output of the following comamnds?

`systemctl status isc-dhcp-server`
`systemctl status tftpd-hpa`

Both daemons are checked to show the status in the dashboard, so maybe is tftpd-hpa the stopped one?

**Comment by doudz** [comment]
```
systemctl status isc-dhcp-server
● isc-dhcp-server.service - ISC DHCP IPv4 server
   Loaded: loaded (/lib/systemd/system/isc-dhcp-server.service; enabled; vendor preset: enabled)
   Active: active (running) since jeu. 2018-09-27 13:44:32 CEST; 1 months 1 days ago
     Docs: man:dhcpd(8)
 Main PID: 11944 (dhcpd)
   CGroup: /system.slice/isc-dhcp-server.service
           └─11944 dhcpd -user dhcpd -group dhcpd -f -4 -pf /run/dhcp-server/dhcpd.pid -cf /etc/dhcp/dhcpd.conf
```
```
systemctl status tftpd-hpa
● tftpd-hpa.service - LSB: HPA's tftp server
   Loaded: loaded (/etc/init.d/tftpd-hpa; bad; vendor preset: enabled)
   Active: active (exited) since mar. 2018-09-04 12:55:28 CEST; 1 months 24 days ago
     Docs: man:systemd-sysv-generator(8)
```

**Comment by doudz** [comment]
I tried `sudo systemctl restart tftpd-hpa.service`
and now everything's fine but it's strange...
![image](https://user-images.githubusercontent.com/1262548/47645245-ae2ffa00-db70-11e8-8f81-650ea7e4baa5.png)

**Comment by jacalvo** [comment]
Ok, I'm going to close this, please don't hesitate to open a new one if you still can provide steps on how to reproduce this on the newly released Zentyal 6.0.
